src/core/Akka.Remote/Transport/TestTransport.cs

```diff
@@ -128,27 +128,25 @@ public override Task<AssociationHandle> Associate(Address remoteAddress)
                 remoteHandle.Writeable = false;
 
                 //pass a non-writeable handle to remote first
-                var remoteAssociationListener = remoteAssociationListenerTask.Result;
+                var remoteAssociationListener = await remoteAssociationListenerTask.ConfigureAwait(false);
```

Comment:
Changed a blocking call here to an `await`-able one so it's closer to being inline with the original Scala source: https://github.com/akka/akka/blob/master/akka-remote/src/main/scala/akka/remote/transport/TestTransport.scala#L60